app/models/mixins/aggregation_mixin.rb

```diff
@@ -8,8 +8,8 @@ module AggregationMixin
     virtual_column :aggregate_vm_cpus,         :type => :integer, :uses => :vms_and_templates
     virtual_column :aggregate_vm_memory,       :type => :integer, :uses => :vms_and_templates
     virtual_column :aggregate_disk_capacity,   :type => :integer, :uses => :hosts
-
-    virtual_column :aggregate_logical_cpus, :type => :integer, :uses => :hosts # Deprecated
+    virtual_column :aggregate_cpu_total_cores, :type => :integer, :uses => :hosts
+    alias_method :aggregate_logical_cpus, :aggregate_cpu_total_cores
```

Comment:
The only problem I see with this is that it doesn't let the user know that this is deprecated.